go-controller/pkg/util/util.go

```diff
@@ -376,3 +376,8 @@ func GenerateId(length int) string {
 	}
 	return string(b)
 }
+
+// IsWildcard checks if the domain name is wildcard.
+func IsWildcard(dnsName string) bool {
+	return strings.HasPrefix(dnsName, "*.")
+}
```

Comment:
Maybe we can store all egress firewall related functions in one file under utils?

Reply:
Moved it to `util/egressfirewall.go`.